### ISSUE TITLE
Delegate user is not valid security principal recipient.

### ISSUE THREAD

**Issue by KristofStroobants** [MicrosoftDocs/office-docs-powershell#11296]
I have 10 room mailboxes. The request is to have one shared mailbox to be used for delegation of agenda items.
So I want to add a shared mailbox as the delegate.

When i try to run  `Add-MailboxFolderPermission -Identity "Room Cacao:\Calendar" -User "Reception@domain.com" -AccessRights Editor -SharingPermissionFlags Delegate, CanViewPrivateItems -SendNotificationToUser $false`

It returns me the error: ` Delegate user 'Company - Reception site X' is not valid security principal recipient.`

However if i manually open a Outlook profile per RoomMailbox, go to files\account settings\delegate, I  **can add this shared mailbox for delegation**.

Any idea how to do this with PowerShell? Or why this does not work?

**Comment by chrisda** [comment]
@KristofStroobants, from the description of the User parameter:

> Valid values are mail-enabled security principals (mail-enabled accounts or groups that have security identifiers or SIDs that can have permissions assigned to them). For example:
>
> - User mailboxes
> - Mail users
> - Mail-enabled security groups

I tried your same commands with Identity and User values customized for my cloud environment. I tried using different identifier values for User to identify the shared mailbox (GUID and DistinguishedName) and I got the following error:

> The user "Sales inquiries" was found in Active Directory but isn't valid to use for permissions. Try an SMTP address instead.

So clearly, PowerShell doesn't like using a Shared mailbox for User. A shared mailbox has an associated user account, but it's disabled with an unknown password. I would guess that if you disable the account for a regular user mailbox and try to use it for the User parameter, you will get the same error.

What's also weird is that `Get-SecurityPrincipal | FT Name` should definitively return any/all objects that are suitable to use for the User parameter, but the shared mailbox that I tried appears in the output. So much for that theory!

For a definitive answer, you could open a support ticket, but the error is pretty clear and the parameter description (intentionally or unintentionally) doesn't claim that shared mailboxes will work with the User parameter.

**Comment by KristofStroobants** [comment]
Hi @chrisda 

Thanks for getting back to me so fast!
Given the fact that it works without PowerShell, i assume this isn't an intended error, perhaps an oversight?
Or perhaps this is a use case that wasn't possible before, i don't really know. Somehow this is the first time I've tried it.
Having said that, we have some more use cases now so perhaps it isn't that weird.

If Outlook was set up so that it wouldn't allow the use of a Shared Mailbox, I'd assume this would throw an error in both worlds, PowerShell and UI Outlook.

As for a small side note, : the Shared Mailbox I've used is/was enabled in AD.
